src/storage/hummock_sdk/src/filter_key_extractor.rs

```diff
@@ -23,9 +24,12 @@ use risingwave_common::util::ordered::OrderedRowDeserializer;
 use risingwave_common::util::sort_util::OrderType;
 use risingwave_pb::catalog::Table;
 use tokio::sync::Notify;
+use tokio::time::timeout;
 
 use crate::key::{get_table_id, TABLE_PREFIX_LEN};
 
+pub static ACQUIRE_TIMEOUT: LazyLock<Duration> = LazyLock::new(|| Duration::from_secs(60));
```

Comment:
`Duration::from_secs` is a `const fn`, so I believe it's okay to make it a constant directly.

```suggestion
const ACQUIRE_TIMEOUT: Duration = Duration::from_secs(60);
```

Reply:
Thanks fix it